src/App.tsx

```diff
@@ -17,39 +56,19 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={query}
+            onChange={e => {
+              setQuery(e.target.value);
+              applyedQuery(e.target.value);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)